src/lightning/pytorch/trainer/connectors/accelerator_connector.py

```diff
@@ -630,10 +630,8 @@ def _lazy_init_strategy(self) -> None:
 
     @property
     def is_distributed(self) -> bool:
-        # TODO: deprecate this property
-        # Used for custom plugins.
-        # Custom plugins should implement is_distributed property.
-        if hasattr(self.strategy, "is_distributed") and not isinstance(self.accelerator, TPUAccelerator):
+        if hasattr(self.strategy, "is_distributed"):
+            # Used for custom plugins. They should implement this property
             return self.strategy.is_distributed
         distributed_strategy = [
```

Comment:
can you pluralize this variable while we're at it
```suggestion
        distributed_strategies = [
```